### PR TITLE
Integration of Login

### PR DESCRIPTION
**Pull request by guilleSequeiraWolox**
## Summary

Add login and integration

## Trello Card

https://trello.com/c/zt9KPAW0/66-login

### CODE REVIEW

File: react/wbooks/src/app/utils/validations.js [hunk 1]
```diff
@@ -16,3 +16,9 @@ export const validationSchemaSignUp = Yup.object().shape({
     .required(i18next.t('Validations:fieldEmpty')),
   password: Yup.string().required(i18next.t('Validations:fieldEmpty'))
 });
+
+export const validationSchemaLogin = Yup.object().shape({
+  email: Yup.string()
+    .email(i18next.t('Validations:invalidEmail'))
+    .required(i18next.t('Validations:fieldEmpty'))
```
Comment (LucasZibell) [inline react/wbooks/src/app/utils/validations.js]: use { t } directly

File: react/wbooks/src/services/AuthServices.js [hunk 2]
```diff
@@ -1,12 +1,20 @@
...
   const userNormalize = normalizeCreateUser(user);
   const userCreated = await serviceCreateUser(userNormalize);
   return userCreated.ok ? { ok: true, data: userCreated.data } : { ok: false, problem: userCreated.problem };
 };
 
+export const login = async user => {
+  const userNormalize = normalizeLogin(user);
+  const userCreated = await serviceLogin(userNormalize);
+  return userCreated.ok ? { ok: true, data: userCreated.data } : { ok: false, problem: userCreated.problem };
+};
```
Comment (LucasZibell) [inline react/wbooks/src/services/AuthServices.js:17]: See what i commented in the other pr

File: react/wbooks/src/app/screens/Login/index.js [hunk 3]
```diff
@@ -1,10 +1,65 @@
...
+      initialValues={initialValuesSignUp}
+      validationSchema={validationSchemaLogin}
+      handleSubmit={onSubmitLogin(state)}
+    >
+      {({ handleSubmit, ...props }) => (
+        <div className={styles.formContainer}>
+          <img className={styles.logoImage} src={logoWolox} />
+          <form className={styles.formContent} onSubmit={handleSubmit}>
+            <CustomField type="email" nameInput="email" nameLabel="Email" {...props} />
+            <CustomField type="password" nameInput="password" nameLabel="Contraseña" {...props} />
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Login/index.js:47]: Why is nameLabel in spanish? Is it the text to be showed on screen?

File: react/wbooks/src/app/screens/Login/index.js [hunk 4]
```diff
@@ -1,10 +1,65 @@
...
+              {i18next.t('LOGIN:login')}
+            </button>
+            <Link className={styles.SignupButton} to="/sign_up">
+              {i18next.t('LOGIN:signUp')}
+            </Link>
+          </form>
+        </div>
+      )}
+    </FormWrapper>
+  );
```
Comment (LucasZibell) [inline react/wbooks/src/app/screens/Login/index.js:58]: Use t

**Review by LucasZibell — CHANGES_REQUESTED** [review]

File: react/wbooks/src/app/screens/Login/index.js (hunk 3 above)
Comment (guilleSequeiraWolox) [reply react/wbooks/src/app/screens/Login/index.js:47]: yes, it is. the text is shows in the screen above of input.

Comment (LucasZibell) [reply react/wbooks/src/app/screens/Login/index.js:47]: You should use i18n 👀

**Review by LucasZibell — APPROVED** [review]

**Review by mariadelacamara — APPROVED** [review]

**Pull request merged**